apps/E2E/src/Checkbox/specs/Checkbox.spec.win.ts

```diff
@@ -27,17 +26,34 @@ describe('Checkbox Accessibility Testing', () => {
   });
 
   it('Checkbox - Validate accessibilityRole is correct', async () => {
```

Comment:
nit same as Button, no need for the "Checkbox - " prefix (same for next two tests)

Reply:
Same for ExperimentalCheckbox